db/models.py

```diff
@@ -1 +1,29 @@
+from typing import ForwardRef
+
 from django.db import models
+from django.db.models import SET_NULL, CASCADE
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField()
```

Comment:
The `description` field in the `Race` model should allow blank values. Consider adding `blank=True` to this field to match the migration file's settings.